### PR TITLE
SAA-1590: Updating dependencies

### PR DESCRIPTION
**Pull request by timharrison-moj**
NPM packages updated to latest minor versions.
Gov uk and MOJ frontend to latest.

### CODE REVIEW

File: backend/setupStaticContent.js [hunk 1]
```diff
@@ -7,9 +7,10 @@ module.exports = () => {
   router.use(express.static(path.join(process.cwd(), '/build')))
 
   const assetPaths = [
```
Comment (timharrison-moj) [inline backend/setupStaticContent.js:9]: Updated paths for the new location of govuk-frontend asssets (within a /dist sub-folder)

File: backend/utils/nunjucksSetup.js [hunk 2]
```diff
@@ -13,7 +13,14 @@ const {
...
-    [path.join(__dirname, '../../views'), 'node_modules/govuk-frontend/', 'node_modules/@ministryofjustice/frontend/'],
+    [
+      path.join(__dirname, '../../views'),
+      'node_modules/govuk-frontend/dist/',
+      'node_modules/govuk-frontend/dist/components/',
+      'node_modules/govuk-frontend/dist/govuk/assets/',
+      'node_modules/@ministryofjustice/frontend/',
+      'node_modules/@ministryofjustice/frontend/components/',
+    ],
     {
```
Comment (timharrison-moj) [inline backend/utils/nunjucksSetup.js:24]: Referencing the new locations for govuk assets

File: cypress.config.ts [hunk 3]
```diff
@@ -16,12 +16,13 @@ export default defineConfig({
...
   videosFolder: 'integration-tests/videos',
+  viewportWidth: 1000,
+  viewportHeight: 1500,
   downloadsFolder: 'integration-tests/downloads',
   reporter: 'cypress-multi-reporters',
   reporterOptions: {
     configFile: 'cypress-reporter-config.json',
   },
-  videoUploadOnPasses: false,
   taskTimeout: 200000,
```
Comment (timharrison-moj) [inline cypress.config.ts:26]: Deprecated

File: cypress.config.ts [hunk 4]
```diff
@@ -58,8 +60,8 @@ export default defineConfig({
           ]),
 
         stubUserEmail: username => auth.stubEmail(username),
-        stubUser: (username, caseload) => auth.stubUser(username, caseload),
-        stubCourts: courts => whereabouts.stubCourts(courts),
+        stubUser: username => auth.stubUser(username),
+        stubCourts: () => whereabouts.stubCourts(),
         stubCourtEmailAddress: court => whereabouts.stubCourtEmailAddress(court),
```
Comment (timharrison-moj) [inline cypress.config.ts:65]: These functions had the wrong number of arguments. No caseload parameter on auth.stubUser() and no courts parameter on whereabouts.stubCourts() - these are statically returned from a JSON file.

File: integration-tests/integration/manageCourts.cy.js [hunk 5]
```diff
@@ -5,6 +5,10 @@ const HomePage = require('../pages/homePage')
 context('A user can view the manage courts page', () => {
   beforeEach(() => {
     cy.task('reset')
+
+    // Added stub - where else would it get its court list from?
+    cy.task('stubCourts')
+
     cy.window().then(win => {
```
Comment (timharrison-moj) [inline integration-tests/integration/manageCourts.cy.js:12]: Could not see where the courts data was stubbed? Added this to provide data required for the test.

File: package.json [hunk 6]
```diff
@@ -9,75 +9,75 @@
...
+    "@types/superagent": "^4.1.24",
+    "@typescript-eslint/eslint-plugin": "^5.62.0",
+    "@typescript-eslint/parser": "^5.62.0",
     "audit-ci": "^6.6.1",
     "concurrently": "^7.6.0",
-    "cypress": "^13.1.0",
-    "cypress-multi-reporters": "^1.6.3",
+    "cypress": "^13.6.6",
+    "cypress-multi-reporters": "^1.6.4",
     "dotenv-expand": "^10.0.0",
```
Comment (timharrison-moj) [inline package.json:60]: Cypress upgraded to most recent version.

File: views/components/timePicker/timePicker.scss [hunk 7]
```diff
@@ -5,6 +5,7 @@
 
   .time-picker-hours, .time-picker-minutes {
     width: 70px;
+    min-width:3em;
   }
```
Comment (timharrison-moj) [inline views/components/timePicker/timePicker.scss:10]: Without this, the time entry fields on the timePicker component don't honour the 70px width - they grow to the govuk-frontend min width.

File: views/partials/layout.njk [hunk 8]
```diff
@@ -40,17 +40,17 @@
...
+  {# Run JavaScript at end of the <body>, to avoid blocking the initial render. #}
+  <script type="module" src="/js/govukFrontendInit.js"></script>
+  <script src="/js/main.js"></script>
+  <script>var MOJFrontend = {};</script>
+
+  {# Javascript block for individual pages #}
+  {% block pageScripts %}
+  {%  endblock %}
+
 {% endblock %}
```
Comment (timharrison-moj) [inline views/partials/layout.njk:52]: Initialise govuk-frontend via a script.

**Review by GurnankCheema — APPROVED** [review]
LGTM

**Pull request merged**